spec/models/activity_spec.rb

```diff
@@ -987,6 +987,20 @@
       end
     end
 
+    describe "#funding_organisation" do
+      let!(:beis) { create(:beis_organisation) }
+
+      it "returns BEIS if the activity is a project" do
```

Comment:
Worth adding cases for programme and third-party project to be on the safe side?